CallsheetLocalizations/it.xcloc/Localized Contents/it.xliff

```diff
@@ -431,482 +515,602 @@
       </trans-unit>
       <trans-unit id="If you play something using [Channels](https://getchannels.com/) or [Plex](https://plex.tv/) on another device, Callsheet can display that show or movie at the top of the Discover screen." xml:space="preserve">
         <source>If you play something using [Channels](https://getchannels.com/) or [Plex](https://plex.tv/) on another device, Callsheet can display that show or movie at the top of the Discover screen.</source>
+        <target state="translated">Se riproduci qualcosa in [Channels](https://getchannels.com/) o [Plex](https://plex.tv/) su un altro dispositivo, Callsheet può mostrare quella serie o film in cima alla schermata Scopri</target>
         <note/>
       </trans-unit>
       <trans-unit id="Image" xml:space="preserve">
         <source>Image</source>
+        <target state="translated">Immagine</target>
         <note/>
       </trans-unit>
       <trans-unit id="In Grace Period" xml:space="preserve">
         <source>In Grace Period</source>
+        <target state="translated">Nel periodo di grazia</target>
         <note>Shown in a semi-hidden debugging view</note>
       </trans-unit>
       <trans-unit id="In Retry Period" xml:space="preserve">
         <source>In Retry Period</source>
+        <target state="translated">Nel periodo di nuova prova</target>
         <note>Shown in a semi-hidden debugging view</note>
       </trans-unit>
       <trans-unit id="In-Credits Bonus Scenes" xml:space="preserve">
         <source>In-Credits Bonus Scenes</source>
+        <target state="translated">Scena bonus nei titoli di coda</target>
         <note/>
       </trans-unit>
       <trans-unit id="Integrations" xml:space="preserve">
         <source>Integrations</source>
+        <target state="translated">Integrazioni</target>
         <note>Header in Persnickety Preferences</note>
       </trans-unit>
       <trans-unit id="JustWatch" xml:space="preserve">
         <source>JustWatch</source>
+        <target state="translated">JustWatch [GuardaEBasta would be terrible]</target>
```

Comment:
This probably should be marked as `Do not translate` since it's referring to a website.